source/backend/dal/Repositories/CompensationRequisitionRepository.cs

```diff
@@ -49,11 +48,9 @@ public PimsCompensationRequisition GetById(long compensationRequisitionId)
                 .Include(x => x.YearlyFinancial)
                 .Include(x => x.ChartOfAccounts)
                 .Include(x => x.Responsibility)
-                .Include(c => c.PimsCompReqH120s)
+                .Include(c => c.PimsCompReqFinancials)
                     .ThenInclude(y => y.FinancialActivityCode)
-                .Include(x => x.PimsAcquisitionPayees)
-                    .ThenInclude(y => y.AcquisitionOwner)
-                .Include(x => x.PimsAcquisitionPayees)
+                .Include(x => x.AcquisitionOwner)
```

Comment:
We need to include here the other potential payee types - InterestHolder, TeamPersonID, etc

Could you please put a TODO here so we don't forget?

Reply:
Not sure is necessary. At this point, the payee side on the front end will simply not work for owners. If the changes require changes in this section of the code they will have to do so